webapp/src/components/host_notices.tsx

```diff
@@ -144,17 +145,24 @@ const RedStyledCompassIcon = styled(StyledCompassIcon)`
 
 export const StyledMonitorAccount = styled(MonitorAccount)<{ $onWidget?: boolean }>`
     flex: none;
-    margin-left: ${({$onWidget}) => ($onWidget ? 1 : 0)}px;
+    margin-left: 0;
     margin-top: 1px;
-    fill: var(--center-channel-color-64);
-    width: ${({$onWidget}) => ($onWidget ? 14 : 18)}px;
+    fill: rgba(var(--calls-bg-rgb), 0.56);
+    width: 18px;
+
+    ${({$onWidget}) => $onWidget && css`
+        fill: var(--center-channel-color-64);
```

Comment:
This would be the first time not using the `rgba()` form. Just raising the inconsistency. I admit it looks cleaner though.

Reply:
Huh, I never even thought about it -- we've been using both in mobile for a long time. If you want I can take it out later in another PR. But personally I don't mind, because it's clear.

Reply:
as an aside -- don't know why I made a ticket for this. I won't include it in my weekly roundup. :p